Client/Frontend/UIConstants.swift

```diff
@@ -53,3 +53,13 @@ public struct UIConstants {
     /// JPEG compression quality for persisted screenshots. Must be between 0-1.
     static let ScreenshotQuality: Float = 0.3
 }
+
+private struct TempStrings {
+    // Bug 1189902 - Offer crash reporting on first run after a crash
+    let crashReportTitle = NSLocalizedString("Oops! Firefox crashed", comment: "Title for prompt displayed to user after the app crashes")
+    let crashReportDescription = NSLocalizedString("Send a crash report so Mozilla can fix the problem?", comment: "Message displayed after the app crashes asking to send crash reports")
+    let sendReport = NSLocalizedString("Send Report", comment: "Send Report option when asking to sending crash reports")
```

Comment:
Based on the ![image](https://bug1189902.bmoattachments.org/attachment.cgi?id=8662500), "Used as a button label for crash dialog prompt" for all four strings should work.